localisation/000_Other_title_names.csv

```diff
@@ -394,8 +394,8 @@ d_sunfury;Sunfury;;;;;;;;;;;;;x
 d_sunfury_adj;Sunfury;;;;;;;;;;;;;x
 e_wastewander;Wastewander;;;;;;;;;;;;;x
 e_wastewander_adj;Wastewander;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+d_darkmoon_island;Darkmoon Island;Darkmoon Island;Darkmoon Island;;Darkmoon Island;;;;;;;;;x
+d_darkmoon_island_adj;Darkmoonian;;darkmoonisch;;;;;;;;;;;x
```

Comment:
Wouldn't it be better if...
```suggestion
d_darkmoon_island_adj;Darkmoon;;darkmoonisch;;;;;;;;;;;x
```
Since you know Darkmoon Island, Darkmoon Faire.

Reply:
I thought of Darkmoon Fair more of a special name, but could go with your version, too.

Reply:
Darkmoon probably works better than Darkmoonian, in my opinion at least. I don't really see any reason why it wouldn't work, and Darkmoonian is a bit awkward to say.